src/App.svelte

```diff
@@ -120,15 +131,19 @@
 
 <main>
 	<div class="row">
-		<div class="column" id="message-log">
+		<div class="column" id="message-log" on:wheel={stopFollow}>
```

Comment:
Simplify this handler and in the button using an arrow function expression

Reply:
do i need to just write it as arrow function   as  `()=>stopFollow()` or directly implement it the function here ?

Reply:
You can assign directly in the arrow function